### PR TITLE
[store][faq] GPX is now supported

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

**Comment by rtsisyk** [comment]
Relevant update of the web-site: https://github.com/organicmaps/organicmaps.github.io/pull/153

File: data/faq.html [hunk 1]
```diff
@@ -3075,33 +3075,33 @@ <h3 lang="tr">Yer imleri nasıl içe aktarılır</h3>
 
       <dd lang="en">
         You can import bookmarks sent from Organic Maps or third party apps if
-        they export bookmarks in KML format:
+        they export bookmarks in KML/KMZ/GPX formats:
```
Comment (biodranik) [inline data/faq.html:3078]: GPX export is not supported yet. Only import.

File: android/src/fdroid/play/listings/zh-TW/full-description.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@
 • 等高線、海拔高度、山峰與谷地
 • 步驟式的行走、單車與實驗性的語言汽車導航
 • 快速的離線搜尋
-• 書籤輸出與匯入 KML/KMZ 格式（GPX 開發中）
+• 書籤輸出與匯入 KML/KMZ/GPX 格式
```
Comment (biodranik) [inline android/src/fdroid/play/listings/zh-TW/full-description.txt:16]: Is it better to replace `KML/KMZ/GPX` everywhere to `KML, KMZ, GPX` for SEO purposes?

File: data/faq.html [hunk 3]
```diff
@@ -3117,21 +3117,21 @@ <h3 lang="tr">Yer imleri nasıl içe aktarılır</h3>
 
       <dd lang="ru">
         Вы можете импортировать метки отправленные из Organic Maps или сторонних
-        приложений, если они позволяют экспортировать метки в формате KML:
+        приложений, если они позволяют экспортировать метки в форматах KML/KMZ/GPX:
```
Comment (biodranik) [inline data/faq.html:3120]: no, do not change text here and in other languages too in this paragraph.

**Review by biodranik — CHANGES_REQUESTED** [review]
@rtsisyk thanks! A few comments.

**Comment by endim8** [comment]
shouldn't it say that it supports importing GPX/KML tracks as well?

**Comment by biodranik** [comment]
> shouldn't it say that it supports importing GPX/KML tracks as well?

Where exactly? Please suggest.

File: data/faq.html (hunk 3 above)
Comment (biodranik) [reply data/faq.html]: Actually, I was wrong, this change looks good.

File: data/faq.html (hunk 1 above)
Comment (biodranik) [reply data/faq.html]: The current text is ok

**Comment by biodranik** [comment]
We can leave the current wording that assumes that GPX export is also implemented, to save time later when we actually implement it and need to change these strings again.

CC @cyber-toad

**Review by biodranik — APPROVED** [review]

**Pull request merged**